client/app/kubernetes/service/kubernetes-service.controller.js

```diff
@@ -93,9 +93,19 @@ angular.module('managerApp').controller('KubernetesServiceCtrl', class Kubernete
     this.ControllerHelper.constructor.downloadContent({ fileContent: this.kubernetesConfig.content, fileName: `${this.kubernetesConfig.fileName}.yml` });
   }
 
+  isProcessing() {
+    return this.cluster.status === 'INSTALLING' || this.cluster.status === 'DELETING' || this.cluster.status === 'RESETTING' || this.cluster.status === 'UPDATING';
```

Comment:
Constants please :)

Reply:
This code looks really similar to https://github.com/ovh-ux/ovh-manager-cloud/pull/1317/files#diff-06c40dc03e83833485b705ad4988ff88R104, I don't know if it makes sense to expose it in a `service` instead

Reply:
Changed to constant and the isProcessing method is moved to a service for reuse